doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -518,7 +518,10 @@ For now, only the value for the ``:DatasetPublishPopupCustomText`` setting from
 
   GET http://$SERVER/api/info/settings/:DatasetPublishPopupCustomText
 
+Get API Terms of Use. The response contains the text value inserted as API Terms use::
```

Comment:
It might be nice to mention the database setting that's used (`:ApiTermsOfUse). Also, "of" is missing. It says "API Terms use".